hack/sync.sh

```diff
@@ -1,6 +1,23 @@
 #!/bin/bash -ex
 
-source ./cluster-up/hack/common.sh
+if [ -z "$KUBEVIRTCI_PATH" ]; then
+    KUBEVIRTCI_PATH="$(
+        cd "$(dirname "$BASH_SOURCE[0]")/"
+        readlink -f ../kubevirtci/cluster-up
+    )"
+fi
+
+
+if [ -z "$KUBEVIRTCI_CONFIG_PATH" ]; then
```

Comment:
`KUBEVIRTCI_CONFIG_PATH` is used by the scripts in `KUBEVIRTCI_PATH`?

Reply:
added comments to explain the environment variables.

Reply:
I don't see the changes we discussed on Monday.
Specifically the comment that explains _why_ `KUBEVIRTCI_CONFIG_PATH` needs to be set in a file that does not make use of it.